inference-engine/samples/benchmark_app/utils.cpp

```diff
@@ -164,6 +165,50 @@ size_t getBatchSize(const benchmark_app::InputsInfo& inputs_info) {
     return batch_size;
 }
 
+InferenceEngine::Layout getLayoutFromString(const std::string& string_layout) {
+    static const std::unordered_map<std::string, InferenceEngine::Layout> layouts = {
+        {"ANY", InferenceEngine::Layout::ANY},
+        {"NCHW", InferenceEngine::Layout::NCHW},
+        {"NHWC", InferenceEngine::Layout::NHWC},
+        {"NCDHW", InferenceEngine::Layout::NCDHW},
+        {"NDHWC", InferenceEngine::Layout::NDHWC},
+        {"OIHW", InferenceEngine::Layout::OIHW},
+        {"C", InferenceEngine::Layout::C},
+        {"CHW", InferenceEngine::Layout::CHW},
+        {"HWC", InferenceEngine::Layout::HWC},
+        {"HW", InferenceEngine::Layout::HW},
+        {"NC", InferenceEngine::Layout::NC},
+        {"CN", InferenceEngine::Layout::CN},
+        {"BLOCKED", InferenceEngine::Layout::BLOCKED}};
```

Comment:
Some of these layouts have no sense from a tool perspective. We should show only layouts which can be applicable to the input.

Reply:
Removed `ANY` and `BLOCKED` layouts, don't know about `C`, is it used in inputs? The others seems applicable.  https://github.com/openvinotoolkit/openvino/pull/7788/commits/dd337477019bf84602bf7117c156882a4e5f821a

Reply:
`C` is valid for input, but `OIHW` seems not

Reply:
Removed https://github.com/openvinotoolkit/openvino/pull/7788/commits/ab0e82e3367c8fb2dee2c6dfd4172b64944748be